test/auth.js

```diff
@@ -36,8 +76,14 @@ describe('auth', function () {
 
     stubOpen = sinon.stub()
     stubOpen.throws('not stubbed')
+
+    tmpNetrc = require('tmp').fileSync().name
```

Comment:
@dickeyxxx gave it a shot, thinking I would try and mock out `~/.netrc` but it caused `require` to fail in `heroku-client` and I could not find a way to make it only override for certain files.

```
  4) auth logout deletes the session & authorization:
     Error: ENOENT, no such file or directory '/Users/rbriggs/src/github.com/heroku/heroku-cli-util/node_modules/heroku-client/lib'
      at Binding.<anonymous> (node_modules/mock-fs/lib/binding.js:1047:13)
      at maybeCallback (node_modules/mock-fs/lib/binding.js:43:17)
      at Binding.lstat (node_modules/mock-fs/lib/binding.js:1044:10)
      at Object.fs.lstatSync (fs.js:961:11)
      at Object.realpathSync (fs.js:1631:21)
      at require (internal/module.js:20:19)
      at Heroku.request (node_modules/heroku-client/index.js:9:19)
      at Heroku.delete (node_modules/heroku-client/index.js:34:17)
      at Object.logout (lib/auth.js:106:38)
      at logout.next (<anonymous>)
      at onFulfilled (node_modules/co/index.js:65:19)
      at node_modules/co/index.js:54:5
      at Object.co (node_modules/co/index.js:50:10)
      at Object.createPromise [as logout] (node_modules/co/index.js:30:15)
      at Context.<anonymous> (test/auth.js:217:17)
```